test/unit/presenters/publishing_api_presenters/detailed_guide_test.rb

```diff
@@ -63,7 +63,7 @@ def present(edition)
           slug: government.slug,
           current: government.current?
         },
-        related_mainstream_content: [],
+        related_mainstream_content: []
```

Comment:
I'm awfully inconsistent about this, as it turns out, unless it's linted against.

I prefer trailing (I added one in this same PR, different file https://github.com/alphagov/whitehall/pull/2580/files/ecd13b32d770abaa95edf0c085337f52bf10fbd9#diff-da11b3576e8a1d61c49f7457c686b02eR26) but I accidentally removed this one.